lib/client.js

```diff
@@ -159,11 +164,6 @@ var Client = function(options) {
   this._context = options.context || null;
   this.ready = false;
 
-  if (this._parent) {
-    this.ready = this._parent.ready;
-    return this; // shortcut handshake
-  }
-
   this.on('app.registered', function(data) {
```

Comment:
so we want to listen to these events now for other instances? are they ever fired in time?
my original thinking was that the metadata should be shared with the parent (since it doesn't change between locations), and the context could be retrieved from the instances API.